sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -15,15 +15,17 @@ namespace Azure.Core
     internal abstract class OperationInternalBase
     {
         private readonly ClientDiagnostics _diagnostics;
-        private readonly string _updateStatusScopeName;
         private readonly IReadOnlyDictionary<string, string>? _scopeAttributes;
         private readonly DelayStrategy? _fallbackStrategy;
         private readonly AsyncLockWithValue<Response> _responseLock;
 
+        private readonly string? _waitForCompletionResponseScopeName;
```

Comment:
I don't like that these properties have become nullable. Later, we have to use `!` operator, meaning that we break our own nullability rules.

Reply:
This is because of this https://github.com/Azure/azure-sdk-for-net/pull/29033#discussion_r891693000

> Would it make sense to set _operationTypeName to null, then, to indicate that we expect it will never be used in a concatenated string?

@annelo-msft so we revert it back to make the defualt as `string.Empty`?

Reply:
Yes.  The reason to have made this nullable instead of an empty string was to ensure that we didn't accidentally print an empty string in the diagnostic scope.  Since you have both convinced me that [this isn't possible](https://github.com/Azure/azure-sdk-for-net/pull/29033#discussion_r891693000), I agree that it is preferred not to make them nullable.